src/import-data/ImportDataDestination.ts

```diff
@@ -68,7 +68,7 @@ const ChoiceButton = (props: ChoiceButtonProps): ReactNode => {
         cursor: disabled ? 'not-allowed' : 'pointer',
       },
       hover: disabled ? undefined : { backgroundColor: colors.accent(0.1) },
-      onClick: !disabled && onClick,
+      onClick: disabled ? undefined : onClick,
```

Comment:
Previously, this would pass `false` to `onClick` when the component was disabled. That doesn't match the type of `onClick`, which is event handler or undefined.